package.json

```diff
@@ -9,7 +9,8 @@
     "install-client": "cd client && yarn install",
     "server": "nodemon server.js --prefix api",
     "client": "npm start --prefix client",
-    "dev": "concurrently \"npm run server\" \"npm run client\""
+    "dev": "concurrently \"npm run server\" \"npm run client\"",
+    "sass": "node-sass -w client/src/css/sass/ -o client/src/css/ --recursive"
```

Comment:
Every time I've used sass, it just worked without the need for any special commands.  What does this actually do and why do we need it?

Reply:
So I have always been under the impression that the sass needed to be compiled down into css...but if that is wrong then we could trim some of this out. The way I set this up we can modularize the .scss files to make it easier to find what you need and every time you save any .scss file it will compile into a single css file